licensing/src/main/java/io/crate/license/Cryptos.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.license;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.InvalidKeyException;
+import java.security.Key;
+import java.security.NoSuchAlgorithmException;
+
+public final class Cryptos {
+
+    private static final String PASSPHRASE = "crate_passphrase";
+    private static final String CIPHER_ALGORITHM = "AES";
+    private static final Key AES_KEY_SPEC = new SecretKeySpec(PASSPHRASE.getBytes(), "AES");
+
+    private Cryptos() {
+    }
+
+    public static byte[] encrypt(byte[] data) {
+        try {
+            Cipher cipher = Cipher.getInstance(CIPHER_ALGORITHM);
+            cipher.init(Cipher.ENCRYPT_MODE, AES_KEY_SPEC);
+            return cipher.doFinal(data);
+        } catch (NoSuchAlgorithmException | InvalidKeyException | NoSuchPaddingException | BadPaddingException
+            | IllegalBlockSizeException e) {
+            throw new IllegalStateException(e);
```

Comment:
is this a relevant exception?

Reply:
what do you mean? 

I think this guards more of a programming error than anything else so I don't think a custom exception is needed.